jobs/build/sweep/Jenkinsfile

```diff
@@ -20,6 +20,11 @@ node {
                 parameterDefinitions: [
                     commonlib.ocpVersionParam('BUILD_VERSION'),
                     commonlib.mockParam(),
+                    booleanParam(
+                        name: 'SWEEP_BUILDS',
+                        defaultValue: true,
+                        description: 'Sweep and attach builds to advisories',
+                    ),
```

Comment:
Why did you do this different than our normal syntax? Like you did in https://github.com/openshift/aos-cd-jobs/pull/2034/files#diff-662d098016d7303f901b7433fb4785f3R67-R72

Reply:
I'd sorta like to go through and change everything to this syntax. But I would avoid mix and match in the same Jenkinsfile.

Reply:
@tbielawa yeah, on the `custom` job i just followed the same format that was already being used ... but apart from `commonlib` calls, the `sweep` job had no other parameters, that's why I already used the declarative style.
But either way is fine to me, we just need to choose one and I can go through our Jenkinsfiles.
Created a story and a note for our next Arch meeting about this topic.
https://jira.coreos.com/browse/ART-1289